express-post-json/index.js

```diff
@@ -14,15 +15,14 @@ app.get('/api/grades', (req, res) => {
 }
 );
 
-app.use(express.json());
-
 app.post('/api/grades', (req, res) => {
-
+  req.body.studentId = nextId;
   grades[nextId] = req.body;
   nextId++;
   res.sendStatus(201);
```

Comment:
Almost. You're still just returning the status. You should also return the new grade object.